tests/e2e/taqueria-plugin-contract-types.spec.ts

```diff
@@ -123,13 +123,13 @@ describe("E2E Testing for taqueria contract types plugin",  () => {
             await exec(`cp e2e/data/timelock.tz ${taqueriaProjectPath}/artifacts`)
             await exec(`cd ${taqueriaProjectPath} && taq generate types`)
             await exec(`cp e2e/data/timelock.ts ${taqueriaProjectPath}/types`)
-            await exec(`cd ${taqueriaProjectPath}/types &&
-                        npm init -y &&
-                        npm install typescript --save-dev &&
-                        npm install @taquito/taquito &&
-                        npm install @taquito/signer &&
-                        npx tsc timelock.ts`)
-            const {stdout, stderr} = await exec(`cd ${taqueriaProjectPath}/types && node timelock.js`)
+            await exec(`cp e2e/data/tsconfig.timelock.json ${taqueriaProjectPath}/types`)
```

Comment:
Should we leave these as individual `exec()` statements, or revert to a compound shell command?

Reply:
I think this way, it's easier to debug which command failed.